include/re_sip.h

```diff
@@ -336,6 +341,7 @@ bool sip_dialog_established(const struct sip_dialog *dlg);
 bool sip_dialog_cmp(const struct sip_dialog *dlg, const struct sip_msg *msg);
 bool sip_dialog_cmp_half(const struct sip_dialog *dlg,
 			 const struct sip_msg *msg);
+const struct uri *sip_dialog_route(const struct sip_dialog *dlg);
```

Comment:
if you want to export this function, the prototype should be removed from `src/sip/sip.h`